runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -189,7 +189,8 @@ private static Dataflow buildMockDataflow(
     return mockDataflowClient;
   }
 
-  private GcsUtil buildMockGcsUtil(boolean bucketExists) throws IOException {
+  private GcsUtil buildMockGcsUtil(Boolean bucketExist, Boolean... bucketExists)
```

Comment:
What is the difference between bucketexist and bucketexists? If the variable names cannot be clarified please add a comment explaining.